src/averageAges.js

```diff
@@ -15,11 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let men = people.filter(person => person.sex === 'm');
+
+  if (arguments.length > 1) {
```

Comment:
just check for century explicitly

Reply:
You mean (century !== undefined) ? I was discussing this check with Dennis Pryadka, and he kinda converted me to arguments.length instead, because it saves from the case when undefined is an actual value assigned to century. Could you please correct me if I'm missing something?